src/main/app/controller/GetRespondentCaseController.ts

```diff
@@ -1,20 +1,21 @@
 import { Response } from 'express';
 
 import { RESPONDENT_TASK_LIST_URL } from '../../steps/urls';
-import { CaseWithId } from '../case/case';
+import { getSystemUser } from '../auth/user/oidc';
+import { CosApiClient } from '../case/CosApiClient';
 
 import { AppRequest } from './AppRequest';
 
 export class GetRespondentCaseController {
   //constructor(protected readonly view: string, protected readonly content: TranslationFn) {}
 
   public async getCase(req: AppRequest, res: Response): Promise<void> {
-    if (req.params?.caseId && req.session?.userCaseList) {
-      req.session?.userCaseList.forEach((element: CaseWithId) => {
-        if (element?.id.toString() === req.params?.caseId) {
-          req.session.userCase = element;
-        }
-      });
+    if (req.params?.caseId) {
+      const caseworkerUser = await getSystemUser();
```

Comment:
Should this be a citizen user and not caseworker?

Reply:
this is currently used across application need to be discussed before removing..